### DISCUSSION TITLE
How to get the all ''2021-07-27 Filecoin Ecosystem Grid'' project's Official website？

### DISCUSSION THREAD

**Discussion by JKinGH** [filecoin-project/community#292] [Q&A]
How to get the all ''2021-07-27 Filecoin Ecosystem Grid'' project's Official website？

Can anyone show a List ?
I found the Information in link: https://github.com/filecoin-project/community/discussions/195 is incomplete.

**Comment by mishmosh** [comment]
We are switching to autogenerating this so it will always have the latest data... should have an update within the next week!
